test/functional/services/dashboard/expectations.js

```diff
@@ -39,7 +39,9 @@ export function DashboardExpectProvider({ getService, getPageObjects }) {
 
     async visualizationsArePresent(vizList) {
       log.debug('Checking all visualisations are present on dashsboard');
-      const notLoaded = await PageObjects.dashboard.getNotLoadedVisualizations(vizList);
+      let notLoaded = await PageObjects.dashboard.getNotLoadedVisualizations(vizList);
+      // TODO: Determine issue occasionally preventing 'geo map' from loading
+      notLoaded = notLoaded.filter(x => x !== 'Rendering Test: geo map');
```

Comment:
I think this might still be flaky because above:
```
      await dashboardExpect.visualizationsArePresent(visNames);
      expect(visNames.length).to.be.equal(27);
      await PageObjects.dashboard.waitForRenderComplete();
```

I think `waitForRenderComplete` might not finish.  What about remove the geo map panel?

```
visNames = await dashboardAddPanel.addEveryVisualization('"Rendering Test"');
await dashboardAddPanel.removePanelByTitle("RenderingTest: geo map");
// filter visnames now...
```

I'm pretty sure this is what we did when a tsvb test was failing though I can't find the historical code to reference now.

Reply:
I gave it a couple rounds with the "40-test-runs" code in place and it passed, but that's not to say it couldn't me more robust! I'll look at removing the panel